mlflow/pyfunc/__init__.py

```diff
@@ -1096,6 +1155,38 @@ def _predict_row_batch(predict_fn, args):
             StringType: np.str_,
         }
 
+        def _convert_array_values(values, spark_type):
```

Comment:
Does this function have to be defined here?

Reply:
It uses `spark_primitive_type_to_np_type` that is defined inside the function

Reply:
Can you pass it?

Reply:
make `spark_primitive_type_to_np_type` to be a global var looks better